### ISSUE TITLE
Copy-Paste Gadget has Unlimited Durability

### ISSUE THREAD

**Issue by fandomaniac** [Direwolf20-MC/BuildingGadgets#134]
The Copy-Paste Gadget has unlimited durability when poweredByFE is set to false.

![copy-paste-bug](https://user-images.githubusercontent.com/37810246/48310811-2cbe6b80-e563-11e8-9735-32112a3ab96b.jpg)

Mod Version: BuildingGadgets v2.4.5
Forge Version: Minecraft Version: Forge 1.12.2-14.23.5.2769

**Comment by fandomaniac** [comment]
No, the config was changed when I added the mod to my existing survival multiplayer server and before any of the tools were crafted. The only other custom settings are that I have disabled both the destruction tool and builders paste, The other active tools have durability working correctly (see below screen grab).

![2018-11-11_01 15 47](https://user-images.githubusercontent.com/37810246/48317984-a1c38c80-e5c7-11e8-81ee-870a8cc9a6eb.png)

**Comment by MichaelHillcox** [comment]
Haha, So it doesn't. I've spotted the issue. I'll get a pull request prepped to address the issue.
